### DISCUSSION TITLE
Semantic Kernel configuration improvments

### DISCUSSION THREAD

**Discussion by akordowski** [microsoft/semantic-kernel#6267] [Q&A] [answered]
I'm currently looking into the [Chat Copilot repo](https://github.com/microsoft/chat-copilot) and have seen that the `KernelMemory` can be configured with `Config` objects. The Semantic `Kernel` is currently missing such an option. Are there any plans to deliver this functionality? If not I would be interested to contribute it to the project.

**Comment by dmytrostruk** [comment, marked as answer]
Hi @akordowski , we have a related task for that: https://github.com/microsoft/semantic-kernel/issues/6081. Feel free to contribute!
